ios/Classes/TiMapViewProxy.m

```diff
@@ -745,6 +745,19 @@ -(void)setPolylines:(id)arg{
     }
 }
 
+-(void)setClusterAnnotation:(id)args
+{
+    ENSURE_DICT(args);
+    NSArray *memberAnn = [args objectForKey:@"memberAnnotations"];
+    id annotation = [args objectForKey:@"annotation"];
```

Comment:
- Rename `memberAnn` to `memberAnnotations` (please check similar cases as well)
- Can't `annotation` be set as a `TiMapAnnotationProxy *` instead of an `id` here? As we know it's expected type already
- Validate the types with `ENSURE_TYPE`

Reply:
If you look in method -(TiMapAnnotationProxy*)annotationFromArg:(id)arg
it accepts NSDictionary also. So annotation should be id.